cpp/unreal_plugins/SimulationController/Source/SimulationController/VehicleAgent.cpp

```diff
@@ -0,0 +1,283 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "SimulationController/VehicleAgent.h"
+
+#include <limits>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <Components/BoxComponent.h>
+#include <Components/PrimitiveComponent.h>
+#include <Components/SceneCaptureComponent2D.h>
+#include <Components/StaticMeshComponent.h>
+#include <Engine/World.h>
+#include <EngineUtils.h>
+#include <GameFramework/Actor.h>
+
+#include "CoreUtils/ArrayDesc.h"
+#include "CoreUtils/Assert.h"
+#include "CoreUtils/Config.h"
+#include "CoreUtils/Log.h"
+#include "CoreUtils/Std.h"
+#include "CoreUtils/Unreal.h"
+#include "SimulationController/CameraSensor.h"
+#include "SimulationController/ImuSensor.h"
+//#include "Vehicle/VehicleMovementComponent.h"
```

Comment:
Why commented?

Reply:
Since we fixed the issue with `Yield` macro, I've refined the code in this class.